### ISSUE TITLE
[REQ] Add support for multiple servers

### ISSUE THREAD

**Issue by unqxl** [Nereziel/cs2-WeaponPaints#178]
Hello, is there a way to support multiple servers in one DB and not using globalshare thing?

**Comment by st1ng2** [comment]
You can use one MySQL database for all servers )

**Comment by unqxl** [comment]
yea, but the problem is that we don't want to have exact same skins on each server

**Comment by unqxl** [comment]
as i saw in database structure, there's no `sid` column in any of table, so players will have same skins

**Comment by L1teD** [comment]
create separate tables for each servers?

**Comment by unqxl** [comment]
Of course, it sounds interesting to create different tables for each server, but the plugin does not provide such an opportunity.
Most of the plugins allow you to replace the value of `sid` in the database, but I do not observe this possibility in this plugin

In general, I have never seen such a thing from anyone using this plugin, but here it would look very good

**Comment by unqxl** [comment]
wdym? maybe players want to have diffrent skins on each server that we have with that plugin?
isn't having `sid` column will solve this problem, like in other plugins?

@Nereziel i wanna hear your opinion, sorry for ping in advance

**Comment by crashzk** [comment]
Support in the plugin for using other tables in the database is not implemented yet. This feature would be good for the future.

Suggestion for now would be to create a new database for each server if you want different skins on each server.

**Comment by unqxl** [comment]
> Suggestion for now would be to create a new database for each server if you want different skins on each server.

yea, we see this as solution for our problem and use it
but we will be happy to see official support for multiple servers in this plugin 😄

**Comment by crashzk** [comment]
I recommend opening the issue again, change the title to **[REQ] Support for multiple servers**

This way, developers can check the suggestion and implement it when they have time.

**Comment by unqxl** [comment]
> I recommend opening the issue again, change the title to **[REQ] Support for multiple servers**

👌

**Comment by daffyyyy** [comment]
Planned
